lib/utils/aws/s3.go

```diff
@@ -45,33 +45,33 @@ func ConvertS3Error(err error, args ...interface{}) error {
 	// SDK v1 errors:
 	var rerr awserr.RequestFailure
 	if errors.As(err, &rerr) && rerr.StatusCode() == http.StatusForbidden {
-		return trace.AccessDenied(rerr.Message())
+		return trace.AccessDenied("%s", rerr.Message())
 	}
 
 	var aerr awserr.Error
 	if errors.As(err, &aerr) {
 		switch aerr.Code() {
 		case s3.ErrCodeNoSuchKey, s3.ErrCodeNoSuchBucket, s3.ErrCodeNoSuchUpload, "NotFound":
-			return trace.NotFound(aerr.Error(), args...)
```

Comment:
This one actually uses `args` in some of the callsites, so we are dropping parts of the error message now.

This is another signature that makes no sense to me.